src/main.c

```diff
@@ -152,15 +153,35 @@ main(int argc, char* argv[])
         { "dump-config", no_argument, 0, 'D' },
         { "rotate", no_argument, 0, 'R' },
         { "config", required_argument, 0, 'f' },
+        { "no-config", required_argument, 0, 'F' },
```

Comment:
Umm, you're totally right!

It seems that depending on the version of getopts required_argument is actually an optional  extra argument or a must. For the gentoo version in libc It forces me to add something.

```
$ sngrep --no-config
-F option requires an argument.
```

If you could update the PR with that change I will appreciate it.

Thanks again!